README.md

```diff
@@ -1986,16 +1987,35 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Line length
+
+  - [26.1](#26.1) <a name='26.1'></a> Avoid very long lines of code.
+
+  > Why? This ensures readability and maintainability.
+
+    ```javascript
+    // bad
+    var foo = { "bar": "This is a bar.", "baz": { "qux": "This is a qux" }, "difficult": "to read" };
```

Comment:
While I don't see a rule in the styleguide about it, in our examples we generally don't quote object keys when they're not invalid identifiers. Can you unquote all of these object keys?